src/common/map/MapService.js

```diff
@@ -120,6 +120,53 @@
     };
   })();
 
+  function bboxTransfom(coordinates, oldProjection, projection) {
```

Comment:
rename to:
`createBBoxFromCoordinatesFromProjectionIntoProjection`
and rename the params to: `fromProjection` `toProjection`